edge/discussion_materials/ONNX Edge scope and profile definition.md

```diff
@@ -0,0 +1,72 @@
+# ONNX Edge scope and profile definition
+Author: Yedong Liu, Milan Oljaca, Saurabh Tangri
+
```

Comment:
I think we should add some preface/overview section, which describes the goal of the document, and the relations between the various things we're trying to define below. Here is my suggestion:
"Motivation:
The following document provides a description of terminology related to "edge" : what is the scope of "edge", what are the scenarios, example to devices which map into the edge, and more. The goal is to define what is contained inside an _**edge profile**_ , based on attributes. 
The next step would be using these definitions to create a _**per profile**_ document which contains:

-  the definition of a profile

- Scenario/s which are relevant to this profile

- Scenario mapping to list of ONNX models (and hence derived ONNX operators)